### ISSUE TITLE
How exactly signatures were produced?

### ISSUE THREAD

**Issue by dremovd** [orcam-public/datahack2019#11]
I tried to reproduce NN feature extraction for all the frames but resulting signatures are worse
Maybe you could provide sample code?

**Comment by orcam-public** [comment]
We used a pretrained resnet50 model from pytorch on (selected frames from) the images file, and saved the output of the last layer prior to the classifier (fully connected) layer. The code is not part of the repo but it’s easy to find similar examples online. Search for “image net fine tuning” plus your preferred deep learning library.

**Comment by dremovd** [comment]
It makes it even more strange (pytorch, head-1, pre-trained resnet50). It's literally what I've done. Do you also rescaling to 224x224 and doing default normalization?

**Comment by orcam-public** [comment]
No. We use the 64x64 crops you got.

**Comment by dremovd** [comment]
FYI:
```
All pre-trained models expect input images normalized in the same way, i.e. mini-batches of 3-channel RGB images of shape (3 x H x W), where H and W are expected to be at least 224. The images have to be loaded in to a range of [0, 1] and then normalized using mean = [0.485, 0.456, 0.406] and std = [0.229, 0.224, 0.225].
```
